src/tools/Avalonia.Designer.HostApp/Avalonia.Designer.HostApp.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFrameworks>net461;netcoreapp2.0</TargetFrameworks>
+    <TargetFrameworks>net461;netcoreapp3.1</TargetFrameworks>
```

Comment:
Not 100% sure we want to change this?

Reply:
Host app TFM should match the TFM of Avalonia.DesktopRuntime. Which is not changed by this PR.